docs/RULES.md

```diff
@@ -1,5 +1,92 @@
-# Game rules
+# Truco Rules
+
+## Game
+
+Truco is a card game played between two players (or in pairs) with a spanish deck (40 cards) wihout 8s, 9s or jokers. There are three ways of scoring points:
+
+* **Flor**: the players compare three cards of the same suits.
+* **Envido**: the players compare two cards of the same suits.
+* **Truco**: the cards are compared individually.
+
+After the **Flor** and **Envido**, the game is divided in three rounds. In each round, each player chooses one of the cards in their hand to compete against the card of the other player. The winner is the one who wins two rounds out of three. 
+
+The player who first reaches 30 points(or 15 points depending the variant of the game) in total wins the match.
+
+## Truco
+
+The players can make a bet called "**Truco**" any time in the game after envido. If **Truco** is called, the players lose the chance to call **Envido**. If there is no bet one point goes to the winner of the round. The players can raise the bet only one time in this order:
+
+* **Truco**: 2 points.
+* **Retruco**: 3 points.
+* **Vale 4**: 4 points.
+
+Players can accept, reject or raise the bet:
+    
+* **accept**: the points in dispute go to the winner of the round.
+* **reject**: the player loses the round and the points go to the other player.
+* **raise**: the player can raise the bet in this sequence.
+    
+    
+        no bet -> truco -> retruco -> quiero vale 4
+    
+## Envido
+
+At the beginning of the round, a player in his turn can call for **envido** then an option will apear for the other player to respond to this bet in any of these ways:
+
+   * **Quiero**: accepts to take the bet.
+   * **Envido**: raise the bet by 2 points.
+   * **Real envido**: raise the bet by 3 points.
+   * **Falta envido**: if accepted the one who wins, wins the game.
+   * **No quiero** - refuses to take the last bet. 
+
+the hierarchy of envido bet is as follows:
+
+        no bet -> envido -> real envido -> falta envido
+
+if a player raises envido bet the other can't go back.
+An example of a envido bet could be like this:
+
+        envido (player 1) -> envido (player 2) -> real envido (player 1) -> quiero (player 2)
+        
+The player who has the best pair will win 7 points (2 + 2 + 3).
+
+Another example with _No quiero_ could be:
+
+        envido (player 1) -> envido (player 2) -> real envido (player 1) -> real envido (player 2) -> falta envido (player 1) -> no quiero (player 2)
+        
+player 1 wins 10 points (2 + 2 + 3 + 3).

```

Comment:
When a player rejects a bet with "no quiero" an additional point goes to the one who raised the bet (+1). Its specified bellow this line but the additional point is not accounted for in this point calculation